README.md

```diff
@@ -113,6 +130,8 @@ nextflow run bam2fastq.nf \
 
 ## Test Data
 
+[ can the test data be added or linked to the repository or download in the Dockerfile? would make testing easier ]
```

Comment:
Not yet. I will have to clean up the BAMs first. Added an issue. The test data in the `test/` directory is from https://github.com/genome/gms/wiki/HCC1395-WGS-Exome-RNA-Seq-Data.